talk/owt/sdk/base/peerconnectiondependencyfactory.cc

```diff
@@ -201,7 +201,6 @@ void PeerConnectionDependencyFactory::
   } else {
     encoder_factory = webrtc::CreateBuiltinVideoEncoderFactory();
   }
-#endif
 
   if (GlobalConfiguration::GetCustomizedVideoDecoderEnabled()) {
```

Comment:
Please confirm you've customized video decoder support for iOS?

Reply:
Custom video decoder is not supported on iOS. But it should work as GetCustomizedVideoDecoderEnabled always returns false on iOS.